data/strings/types_strings.txt

```diff
@@ -6896,6 +6896,13 @@
     vi = Ẩm thực nhật bản
     zh-Hans = 日本菜
     zh-Hant = 日本料理
+    
+      [type.cuisine.juice]
+    en = Juice
```

Comment:
Please generate appropriate translations for ALL languages using tools in this order (for missing translations):
1. Wikipedia
2. DeepL (it has pt-BR and pt)
3. Google Translate

Reply:
For this, would I be able to use the [category language generator](https://github.com/organicmaps/organicmaps/pull/3132#discussion_r940705151) for this and copy the code accordingly?

Reply:
This is a good start. Then you can check/correct it using other tools.